tests/Resources/Page/HybridCloudConsole/Rhosak.robot

```diff
@@ -189,3 +191,8 @@ Clean Up RHOSAK
   Capture Page Screenshot  after deleting_stream.png
   Click Link  Service Accounts
   Delete Service Account By Client ID  client_id_delete=${sa_clientid_to_delete}
+  Switch Window  title:Red Hat OpenShift Data Science Dashboard
+  Close All Browsers
+  Launch Dashboard  ocp_user_name=${TEST_USER.USERNAME}  ocp_user_pw=${TEST_USER.PASSWORD}  ocp_user_auth_type=${TEST_USER.AUTH_TYPE}
+  ...               dashboard_url=${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  browser_options=${BROWSER.OPTIONS}
```

Comment:
In line 194- you are going back to RHODS windows.
In line 195 - you are closing the browser
In line  196~197 -> This keyword open new browser and does login

I don't understand why you have to close and relaunch again? am i missing anything?

After Switch window can't we use keyword in 198 . it could relaunching of dashboard

Reply:
I know it sounds meaningless, but it helps in make the TC quicker because of the Dashboard refresh mechanism.We spare up to 1 min I think

Reply:
Hmmm..if we are closing the browser, i don't think we have to Switch to RHODS dashboard(line 194)

Reply:
hmm that's right, it was there from prev version of the TC. I can remove it

Reply:
done